lib/need_for_type/states/game.rb

```diff
@@ -105,21 +110,27 @@ def handle_in_game_valid_input
     def handle_in_game_invalid_input
       Curses.beep
 
-      @display_window.render_game_text(@text, @chars_completed, true)
+      @display_window.render_game_text(@text, @chars_completed, @stats, true)
 
       @state = :in_game_get_input
 
       return self
     end
 
     def handle_end_game
-      @end_time = Time.now
+      calculate_stats
+
+      return NeedForType::States::End.new(@display_window,
```

Comment:
we can have this one liner no need for multiline I think. Just being picky here 😛